### PR TITLE
Update favicon to be CRT logo

### PR DESCRIPTION
**Pull request by maxslarsson**

### CODE REVIEW

**Comment by maxslarsson** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/81?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#81** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#80** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/80?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#79** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/79?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @maxslarsson and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: ground-server/src/app/apple-icon.png
Comment (ZachGarcia42) [inline ground-server/src/app/apple-icon.png:1]: Why apple-icon?

File: ground-server/src/app/favicon.ico
Comment (ZachGarcia42) [inline ground-server/src/app/favicon.ico:1]: Why does this also show the old next favicon?

**Review by ZachGarcia42 — APPROVED** [review]
Couple questions, but the site looks good!

**Review by ZachGarcia42 — COMMENTED** [review]
Also quick workflow note. Since this doesn't depend on #80, you could base it off #79 instead. Then you wouldn't have to wait for #80 to merge

**Comment by maxslarsson** [comment]
Good call, I didn't think of that! I'll do that in the future

File: ground-server/src/app/favicon.ico
Comment (maxslarsson) [reply ground-server/src/app/favicon.ico:1]: I think it shows old favicon on left and new on right? That is if you're seeing what I am seeing.

File: ground-server/src/app/apple-icon.png
Comment (maxslarsson) [reply ground-server/src/app/apple-icon.png:1]: My understanding is Apple needs a higher resolution than the favicon (which is 48x48 pixels) for things like the touch bar, Safari, if someone adds the website to the iphone home screen as an app, etc. Next has built in support for it so I figured I might as well add it.

Comment (ZachGarcia42) [reply ground-server/src/app/apple-icon.png:1]: Ah, sweet!

File: ground-server/src/app/favicon.ico
Comment (ZachGarcia42) [reply ground-server/src/app/favicon.ico:1]: Yeah, that makes sense!

**Comment by maxslarsson** [comment]
### Merge activity

* **Oct 1, 12:15 PM EDT**: @maxslarsson started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/81).

**Pull request merged**